vnext/Shared/tracing/cmd.txt

```diff
@@ -0,0 +1,7 @@
+REM Enable RNW provider/collector along side general system event providers.
+wpr.exe -start GeneralProfile -start rnw.wprp
+wpr.exe -start GeneralProfile -start rnw.etl
+
+REM Enable only RNW provider/collector.
+wpr.exe -start rnw.wprp
+wpr.exe -stop rnw.etl
```

Comment:
consider moving this to a PS script in vnext\scripts

Can you add a small how-to in the wiki?

Reply:
Hi @asklar,

I've now added PS scripts in vnext\scripts\Tracing\ to manage the sessions.

Regarding the how-to documentation, where should I add the information ? Is is under top-level docs folder in the repo ? Should i update an existing file or add a new "tracing.md" ?

Thanks!

Reply:
great thanks! we have some docs under \docs and we also have a wiki:
![image](https://user-images.githubusercontent.com/22989529/104692594-179e7380-56bd-11eb-9a8b-c3393f36835b.png)

I can see the how-to becoming bigger as we continue to invest in perf measurement and tooling, and eventually make this available/easily consumable to 3rd parties, so we will probably want to _eventually_ make it part of the docs that live on the RNW website (https://microsoft.github.io/react-native-windows/) but for now it's fine to keep in the wiki (or docs if you prefer), and we can move it once it's meaty enough

Reply:
Added a rudimentary documentation to start with .. https://github.com/microsoft/react-native-windows/blob/master/docs/tracing.md ..